src/content/docs/en/guides/integrations-guide/alpinejs.mdx

```diff
@@ -106,7 +106,7 @@ import alpine from '@astrojs/alpinejs';
 
 export default defineConfig({
   // ...
-  integrations: [alpine({ entrypoint: '/src/entrypoint' })],
```

Comment:
In this case, it's right since the default import is being named `alpine`  here, but since that would be different with the example above, let's change that as well!